Installation/changes.html

```diff
@@ -241,7 +241,12 @@ <h3>Linear cell complex</h3>
     </li>
   </ul>
 
-  <h3> 2D Triangulation data structure </h3>
+ <h3>Range support</h3>
```

Comment:
A bit late but this should have been Arrangement...

Reply:
Commit directly in master

Reply:
It never made it to changes in 4.11. Not a big deal.